blocks/bricks/sequence_generators.py

```diff
@@ -136,6 +137,34 @@ def _push_allocation_config(self):
 
     @application
     def cost(self, application_call, outputs, mask=None, **kwargs):
+        """Returns sum of generation costs for output sequences.
+
+        Parameters
+        ----------
+        outputs : :class:`~tensor.TensorVariable`
+            The 3(2) dimensional tensor containing output sequences.
+            The dimension 0 must stand for time, the dimension 1 for the
```

Comment:
I just noticed that this is a direct copy of the `cost_matrix` docstring where these deficiencies originate. You can fix them if you like, but even if not, for now, I'd rather that we didn't duplicate identical documentation for two methods.

@bartvm I looked at the metadocumentation and couldn't find a final word on this. Should we handle this in `cost_matrix` with a 

```
Notes
-----
See :meth:`cost` documentation for parameters.
```
or something like that?